protobuf/answer.proto

```diff
@@ -26,8 +26,8 @@ package grakn.protocol;
 
 message Answer {
     oneof answer {
-        AnswerGroup answerGroup = 1;
-        ConceptMap conceptMap = 2;
+        AnswerGroup answer_group = 1;
+        ConceptMap concept_map = 2;
```

Comment:
In order for gRPC Java to assign sensible method names, we need to enforce a consistent casing convention. The preferred convention for protocol buffers is snake_case.